### ISSUE TITLE
location update but connection failed

### ISSUE THREAD

**Issue by ramu369** [tananaev/traccar-client-android#121]
hai
i am trying to traccar in windows system.configured also.in my mobile also installed traccar client.shown status connection failed.please help me..

**Comment by tananaev** [comment]
Check if the port is open:
http://www.yougetsignal.com/tools/open-ports/

**Comment by a42** [comment]
Hi there

I have checked which ports my ISP has opened for (with http://mxtoolbox.com/SuperTool.aspx ), and have according to the result (port 8080 open) changed the config for that bit:  


    <entry key='http.enable'>true</entry>
   
    <entry key='http.port'>8080</entry>
    <entry key='http.application'>/opt/traccar/traccar-web.war</entry>

and set the port on the android tracker client to be 8080 but I also get connection fail...

What to do ?

Cheers,
Adam

**Comment by tananaev** [comment]
You changed web interface port. For Traccar Client you need to use port 5005.

**Comment by a42** [comment]
Yes, I did change that :) and now I just realised that all I had to do was to create some port forwarding on my router ...

So now it's working on a local machine, next I want to make it work on a cloud server. :)

Cheers,
Adam

**Comment by tananaev** [comment]
Note that to run Traccar you need a VPS. Regular web hosting is not enough.

**Comment by a42** [comment]
Ok. Point taken :) I'm using Linode as my provider so in that way it should be okay...

Cheers,
Adam
